projects/1.16/assets/biomancy/biomancy/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+{
+  "itemGroup.biomancy": "血肉重铸",
+  "key.categories.biomancy": "血肉重铸",
+  "key.biomancy.item_default": "切换模式",
+
+  "item.biomancy.unveiling_oculi": "揭示之复眼",
+  "item.biomancy.empty_bag": "小胃囊",
+  "item.biomancy.single_item_bag": "物品胃囊",
+  "item.biomancy.entity_storage": "实体胃囊",
+  "item.biomancy.oculus_key": "眼球钥匙",
+  "item.biomancy.injection_device": "注射枪",
+  "item.biomancy.reagent": "试剂",
+  "item.biomancy.fleshborn_pickaxe": "血肉镐",
+  "item.biomancy.fleshborn_axe": "血肉斧",
+  "item.biomancy.fleshborn_shovel": "血肉锹",
+  "item.biomancy.sharp_bone": "锋利骨剑",
+  "item.biomancy.leech_claw": "水蛭之爪",
+  "item.biomancy.long_range_claw": "细长指爪",
+  "item.biomancy.fleshborn_war_axe": "枯萎战斧",
+  "item.biomancy.fleshborn_sword": "贪婪之刃",
+  "item.biomancy.fleshborn_guan_dao": "肌腱关刀",
+  "item.biomancy.withershot": "凋灵枪",
+  "item.biomancy.tooth_gun": "牙枪",
+  "item.biomancy.boomling": "爆虫",
+  "item.biomancy.boomling_hive_gun": "[WIP]爆虫枪",
+  "item.biomancy.bee_hive_gun": "[WIP]蜜蜂枪",
+  "item.biomancy.sewing_kit": "骨针和线",
+  "item.biomancy.sewing_kit_empty": "骨针",
+  "item.biomancy.glass_vial": "玻璃试管",
+
+  "item.biomancy.nutrient_slurry_bucket": "营养液桶",
+  "item.biomancy.mutagenic_bile": "诱变胆汁",
+  "item.biomancy.rejuvenating_mucus": "复原粘液",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "item.biomancy.rejuvenating_mucus": "复原黏液",

```
到底是“黏液”，还是“粘液”？如何取舍？_ni
https://www.sohu.com/a/295341566_681842

---

了解了。顺便一提mucus在参考词典上有粘液，来源是morebees模组。